argoverse/utils/sim2.py

```diff
@@ -156,7 +156,7 @@ def save_as_json(self, save_fpath: _PathLike) -> None:
         """
         if not Path(save_fpath).exists():
```

Comment:
Hi @jlambert,

Thanks for adding this. I notice that this function calls `save_json_dict`. Do you think that it might be convenient if we move directory creation into that function?

My thought is this: https://github.com/argoai/argoverse-api/blob/9ad7251e863590603c3f6064a45830dabd3a15e2/argoverse/utils/json_utils.py#L32 could be modified as:

```
json_fpath = Path(json_fpath)
json_fpath.parent.mkdir(parents=True, exist_ok=True)
with json_fpath.open("w") as f:
    json.dump(dictionary, f)

```

`Path(save_fpath).parent.mkdir(parents=True, exists_ok=True)`

Reply:
+1 on this.